### PR TITLE
spire: fix GHSA-vfp6-jrw2-99g9

### PR DESCRIPTION
**Pull request by hectorj2f**
```
wolfictl scan  packages/aarch64/spire-agent-1.8.3-r1.apk --govulncheck
🔎 Scanning "packages/aarch64/spire-agent-1.8.3-r1.apk"
✅ No vulnerabilities found
```

### CODE REVIEW

File: spire-server.yaml [hunk 1]
```diff
@@ -47,6 +47,9 @@ pipeline:
       echo ${goversion} > .go-version
       cat .go-version
 
+      go get github.com/sigstore/cosign/v2@v2.2.1
```
Comment (pdeslaur) [inline spire-server.yaml:50]: If possible, could you leave a comment with the CVE ID: CVE-2023-46737

**Comment by pdeslaur** [comment]
> ```
> wolfictl scan  packages/aarch64/spire-agent-1.8.3-r1.apk --govulncheck
> 🔎 Scanning "packages/aarch64/spire-agent-1.8.3-r1.apk"
> ✅ No vulnerabilities found
> ```

The scan above is for `1.8.3-r1`, but this PR is `1.8.4-r1` I wonder if we need the fix in the first place? 🤔

**Comment by hectorj2f** [comment]
>  The scan above is for 1.8.3-r1, but this PR is 1.8.4-r1 I wonder if we need the fix in the first place? 🤔

No, that was me wrongly pasting a scan result of an older version that I previously patched. Once I rebased the latest version 1.8.4, I found the same results as you can see in the pushed yaml.

```
wolfictl scan  packages/aarch64/spire-agent-1.8.4-r1.apk --govulncheck
🔎 Scanning "packages/aarch64/spire-agent-1.8.4-r1.apk"
✅ No vulnerabilities found
```

File: spire-server.yaml (hunk 1 above)
Comment (hectorj2f) [reply spire-server.yaml:51]: Added

**Review by pdeslaur — APPROVED** [review]

**Comment by hectorj2f** [comment]
@pdeslaur Thanks for the review.

**Pull request merged**